### PR TITLE
Template English Wiktionary

### PR DESCRIPTION
**Pull request by Ladsgroup**
We decided to make the labeling campaign from scratch ^_^

Diff: https://gist.github.com/Ladsgroup/3efbe704382b3299c00bb7be03dbfb2f
Note that wikidata part is just there to show the differences, nothing has been implemented for wikidata (yet)

### CODE REVIEW

**Comment by halfak** [comment]
The diff for this is confusing.  I see some huwiki stuff.  Is that intentional?

**Comment by Ladsgroup** [comment]
Yup, they are leftovers from #135

File: Makefile [hunk 1]
```diff
@@ -631,6 +633,79 @@ enwiki_tuning_reports: \
...
+datasets/enwiktionary.revisions_for_review.5k_2016.json: \
+		datasets/enwiktionary.autolabeled_revisions.200k_2016.json
+	( \
+	 cat $< | \
+	 grep '"needs_review": true' | \
+	 shuf -n 2500; \
+	 cat $< | \
+	 grep '"needs_review": false' | \
+	 shuf -n 2500 \
+	) | shuf > $@
```
Comment (halfak) [inline Makefile]: If we're restarting, let's not do this at all.

Comment (Ladsgroup) [reply Makefile]: Done!

**Comment by halfak** [comment]
How many observations do you get in the unbalanced "needs_review" sample?

**Comment by Ladsgroup** [comment]
(p3) ladsgroup@stat1005:~/editquality$ wc -l datasets/enwiktionary.revisions_for_review.5k_2016.json 
13626 datasets/enwiktionary.revisions_for_review.5k_2016.json
:/

**Comment by halfak** [comment]
We should probably cut the initial sample size down to 92k.  That will get us ~5k revisions needing review.  I wonder how that will affect the reverted model.

**Comment by Ladsgroup** [comment]
What do you think if I downsize the need to review sample using shuf from 13K to 5K and in order to make the damaging model (when it got done), we get a downsized sample of edits that don't need to review (from the 200k sample) and merge it with the 5k sample?

**Comment by halfak** [comment]
@Ladsgroup, let's downsize the needs-review sample, but not the not-needs review.  The proportion difference is OK.  We can re-adjust stats using the pop-rate params anyway.

**Comment by Ladsgroup** [comment]
Done!

**Comment by halfak** [comment]
Note that we're gonna have a weird template for this dataset.  Oh well.

File: Makefile [hunk 2]
```diff
@@ -670,6 +670,72 @@ enwiki_tuning_reports: \
...
+	grep '"needs_review": true' $< | shuf -n 5000 > $@
+
+datasets/enwiktionary.autolabeled_revisions.w_cache.20k_2016.json: \
+		datasets/enwiktionary.autolabeled_revisions.200k_2016.json
+	shuf -n 20000 $< | \
+	revscoring extract \
+		editquality.feature_lists.enwiktionary.reverted \
+		--host https://en.wiktionary.org \
+		--extractor $(max_extractors) \
+		--verbose > $@
```
Comment (halfak) [inline Makefile:855]: I see now that we're not training on the 200k observations at all.  Why did we sample 200k again?  I can't figure it out.  It seems like we should just use 92k observations for everything and call it good.  Is there something else that I am missing?

Comment (Ladsgroup) [reply Makefile:855]: I doubled checked and it seems you're not missing anything.

**Review by adamwight — APPROVED** [review]
92k changes look good.

**Pull request merged**